packages/transform-eventual-send/package.json

```diff
@@ -7,20 +7,13 @@
     "test": "tape -r esm 'test/**/*.js'",
     "lint-fix": "eslint --fix '**/*.js'",
     "lint-check": "eslint '**/*.js'",
-    "build": "node -r esm ./scripts/build-esend.js"
+    "build": "exit 0"
```

Comment:
This is vestigial. I assume this needs to be here so monorepo build works for every workspace?

I like `true` here.

Reply:
Yeah, we started using `exit 0` in the (small) subset of packages that *didn't* need a build step, but at this point there's only one package left that *does* need one (`wallet-frontend`). We should look into it later, I think we might be able to delete the top-level `yarn build` script entry and all of these no-op `build` lines too.